envs/monkey_zoo/docs/fullDocs.md

```diff
@@ -0,0 +1,1096 @@
+This document describes Infection Monkey’s test network, how to deploy and use it.
+
+[Warning\!](#warning)<br>
+[Introduction](#introduction)<br>
+[Getting started](#getting-started)<br>
+[Using islands](#using-islands)<br>
+[Running tests](#running-tests)<br>
+[Machines’ legend](#machines-legend)<br>
+[Machines](#machines)<br>
+[Nr. 2 Hadoop](#_Toc526517182)<br>
+[Nr. 3 Hadoop](#_Toc526517183)<br>
+[Nr. 4 Elastic](#_Toc526517184)<br>
+[Nr. 5 Elastic](#_Toc526517185)<br>
+[Nr. 6 Sambacry](#_Toc536021459)<br>
+[Nr. 7 Sambacry](#_Toc536021460)<br>
+[Nr. 8 Shellshock](#_Toc536021461)<br>
+[Nr. 9 Tunneling M1](#_Toc536021462)<br>
+[Nr. 10 Tunneling M2](#_Toc536021463)<br>
+[Nr. 11 SSH key steal](#_Toc526517190)<br>
+[Nr. 12 SSH key steal](#_Toc526517191)<br>
+[Nr. 13 RDP grinder](#_Toc526517192)<br>
+[Nr. 14 Mimikatz](#_Toc536021467)<br>
+[Nr. 15 Mimikatz](#_Toc536021468)<br>
+[Nr. 16 MsSQL](#_Toc536021469)<br>
+[Nr. 17 Upgrader](#_Toc536021470)<br>
+[Nr. 18 WebLogic](#_Toc526517180)<br>
+[Nr. 19 WebLogic](#_Toc526517181)<br>
+[Nr. 20 SMB](#_Toc536021473)<br>
+[Nr. 21 Scan](#_Toc526517196)<br>
+[Nr. 22 Scan](#_Toc526517197)<br>
+[Nr. 23 Struts2](#_Toc536021476)<br>
+[Nr. 24 Struts2](#_Toc536021477)<br>
+[Nr. 250 MonkeyIsland](#_Toc536021478)<br>
+[Nr. 251 MonkeyIsland](#_Toc536021479)<br>
+[Network topography](#network-topography)<br>
+
+# Warning\!
+
+This project builds an intentionally
+<span class="underline">vulnerable</span> network. Make sure not to add
+production servers to the same network and leave it closed to the
+public.
+
+# Introduction:
+
+MonkeyZoo is a Google Cloud Platform network deployed with terraform.
+Terraform scripts allows you to quickly setup a network that’s full of
+vulnerable machines to regression test monkey’s exploiters, evaluate
+scanning times in a real-world scenario and many more.
+
+# Getting started:
+
+Requirements:
+1.  Have terraform installed.
+2.  Have a Google Cloud Platform account (upgraded if you want to test
+    whole network at once).
+
+To deploy:
+1.  Configure service account for your project:
+
+    a. Create a service account and name it “your\_name-monkeyZoo-user” 
+    
+    b. Give these permissions to your service account:
+    
+    **Compute Engine -> Compute Network Admin**
+    and
+    **Compute Engine -> Compute Instance Admin**
```

Comment:
My mistake, we also require
Compute Engine -> Compute Network Admin
Compute Engine -> Service Account User